cloud-chatops/tests/test_find_prs.py

```diff
@@ -8,6 +8,9 @@
 from pr_dataclass import PR
 
 
+# pylint: disable=R0801
```

Comment:
Should this line be removed as the pylintc file does not have this code in it anymore?